MoEngagePluginBase.podspec

```diff
@@ -22,6 +22,6 @@ Pod::Spec.new do |s|
     s.source_files = 'MoEngagePluginBase/**/*'
     s.swift_version = '5.0'
     s.frameworks = 'UIKit', 'Foundation', 'UserNotifications'
-    s.dependency 'MoEngage-iOS-SDK', '>= 9.15.0', '< 9.16.0'
-    s.dependency 'MoEngageInApp', '>= 4.14.0', '< 4.15.0'
+    s.dependency 'MoEngage-iOS-SDK', '>= 9.16.0', '< 9.17.0'
```

Comment:
Shouldn't the current pod version be updated as well? Please do the same for other modules.

Reply:
@msoumya-engg-sdk  the release version and date should we updated at the time of release, after merging to master.. just like we do for native

Reply:
Shouldn't the dependency be updated after merging to `master` in that case? Currently `MoEngagePluginBase` dependency is mentioned `4.7.0` for other modules while `MoEngagePluginBase` version is kept as `4.6.0`.